ldk/javascript/src/whisper/react/renderer-config.ts

```diff
@@ -0,0 +1,337 @@
+    type: Type,
+    oldProps: Props,
+    newProps: Props,
+    internalInstanceHandle: any,
+    keepChildren: boolean,
+    recyclableInstance: null | Instance,
+  ): Instance;
+
+  createContainerChildSet(container: Container): ChildSet;
+
+  appendChildToContainerChildSet(childSet: ChildSet, child: Instance | TextInstance): ChildSet;
+
+  finalizeContainerChildren(container: Container, newChildren: ChildSet): void;
+
+  replaceContainerChildren(container: Container, newChildren: ChildSet): void;
+
+  getOffscreenContainerType(): string;
+
+  getOffscreenContainerProps(mode: any, children: any): Props;
+
+  cloneHiddenInstance(
+    instance: Instance,
+    type: Type,
+    props: Props,
+    internalInstanceHandle: any,
+  ): Instance;
+
+  cloneHiddenTextInstance(
+    instance: Instance,
+    text: string,
+    internalInstanceHandle: any,
+  ): TextInstance;
+}
+
+export const config: CoreConfig & PersistenceConfig = {
+  afterActiveInstanceBlur: undefined,
+  appendChildToContainerChildSet(childSet: ChildSet, child: Instance | TextInstance): ChildSet {
+    if (child.type !== 'whisper') {
+      throw new Error('oh-whisper must be top-level element');
+    }
+    childSet.label = child.label;
+    childSet.components = child.components as Component[];
+    return childSet;
+  },
+  appendInitialChild(parentInstance: Instance, child: Instance | TextInstance): void {
+    const result =
+      getHandlerByHelpsType(parentInstance.type).appendInitialChild?.(parentInstance, child) ||
+      false;
+    if (!result) {
+      console.warn(
+        `Cannot add item of type ${child.type} to parent of type ${parentInstance.type}`,
+      );
+    }
+  },
+  beforeActiveInstanceBlur: undefined,
+  cancelTimeout(id: HostConfigTimeoutHandle): void {
+    clearTimeout(id);
+  },
+  cloneHiddenInstance(
+    instance: Instance,
+    type: Type,
+    props: Props,
+    internalInstanceHandle: any,
+  ): Instance {
+    throw new Error('Not Implemented');
+  },
+  cloneHiddenTextInstance(
+    instance: Instance,
+    text: string,
+    internalInstanceHandle: any,
+  ): TextInstance {
+    throw new Error('Not Implemented');
+  },
+  cloneInstance(
+    instance: Instance,
+    updatePayload: Update | null,
+    type: Type,
+    oldProps: Props,
+    newProps: Props,
+    internalInstanceHandle: any,
+    keepChildren: boolean,
+    recyclableInstance: Instance | null,
+  ): Instance {
+    const value: any = {
+      type: instance.type,
+      ...newProps,
+    };
+    delete value.children;
+    if (!keepChildren && value.components) {
+      value.components = [];
+    }
+    if (internalInstanceHandle.key) {
+      value.key = internalInstanceHandle.key;
+    }
+    getHandlerByTagType(type).assignTextChildren?.(value, newProps);
+    return value;
+  },
+  createContainerChildSet(container: Container): ChildSet {
+    return {
+      label: '',
+      components: [],
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
this is pretty minor, but documentation recommends to do:
`
onclose: () => {
  // do nothing.
}
`
https://eslint.org/docs/rules/no-empty-function